### DISCUSSION TITLE
Would like to know how to setup the lazyload for the website

### DISCUSSION THREAD

**Discussion by saizdx2010** [uasoft-indonesia/badaso#769] [General]
I'm currently using the Badaso menit theme where there is a lazyload implementation, but there is a slight problem with the lazyload where it takes a bit too long to load the website's content. So I would like to know is there a function or a method that you guys are using for implementing the lazyload or how to setup the lazyload itself, Thank you.
![image](https://user-images.githubusercontent.com/69862759/185862394-6d81e2f8-c100-401e-b2ec-0f26bf967985.png)

**Comment by rizkiheryandi** [comment]
Hello @saizdx2010 

Thanks for using badaso menit theme

The latest version of badaso menit theme already has lazy load content. 

Please update it to 1.0.0-alpha.2

Sorry for late response
